packages/webapp/src/routes/credits-purchase/credits-metric-upload-check.js

```diff
@@ -0,0 +1,54 @@
+import path from 'path'
+import { getHumanReadableFileSize } from '../../utils/helpers.js'
+import { deleteBlobFromContainers } from '../../utils/azure-storage.js'
+import creditsPurchaseConstants from '../../utils/credits-purchase-constants.js'
+
+const href = '#check-upload-correct-yes'
+const handlers = {
+  get: async (request, h) => {
+    const context = getContext(request)
+    return h.view(creditsPurchaseConstants.views.CREDITS_PURCHASE_CHECK_UPLOAD_METRIC, context)
+  },
+  post: async (request, h) => {
+    const checkUploadMetric = request.payload.checkUploadMetric
+    const metricUploadLocation = request.yar.get(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_METRIC_LOCATION)
+    request.yar.set(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_METRIC_FILE_CHECKED, checkUploadMetric)
+    if (checkUploadMetric === creditsPurchaseConstants.options.creditsCheckUploadMetric.NO) {
+      await deleteBlobFromContainers(metricUploadLocation)
+      request.yar.clear(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_METRIC_LOCATION)
+      return h.redirect(creditsPurchaseConstants.routes.CREDITS_PURCHASE_UPLOAD_METRIC)
+    } else if (checkUploadMetric === creditsPurchaseConstants.options.creditsCheckUploadMetric.YES) {
+      return h.redirect(creditsPurchaseConstants.routes.CREDITS_PURCHASE_CONFIRM_DEV_DETAILS)
+    }
+    return h.view(creditsPurchaseConstants.views.CREDITS_PURCHASE_CHECK_UPLOAD_METRIC, {
+      filename: path.basename(metricUploadLocation),
+      ...getContext(request),
+      err: [
+        {
+          text: 'Select yes if this is the correct file',
```

Comment:
Should be `Select yes if this is the statutory biodiversity metric file your local planning authority reviewed with your biodiversity net gain statement` to match the copy deck